inc/ticketconfig.class.php

```diff
@@ -35,7 +35,10 @@
 
 class PluginCreditTicketConfig extends CommonDBTM {
 
-   public static $rightname = 'entity';
+   public static $rightname = 'plugin_creditticketconfig';
+
+   const TICKET_TAB  = 1024;
+   const TICKET_FORM = 2048;
```

Comment:
A migration should be added to add `plugin_creditticketconfig:TICKET_TAB&TICKET_FORM` right when `plugin_creditticketconfig` does not exist yet and user has the `entity:UPDATE` right.

The `Migration::addRight()` method could probably be used for this.